### PR TITLE
bidsConcatBetaTmaps hoxfit for ffxDir definition in subfun

### PR DESCRIPTION
**Pull request by CerenB**
Here it comes a hot-fix! 
I added `ffxDir` folder in the subfolder to be called. otherwise `remove-useless-betas-subfun` is crashing. 
Should fix the issue #293

### CODE REVIEW

File: src/workflows/bidsConcatBetaTmaps.m [hunk 1]
```diff
@@ -65,15 +65,16 @@ function bidsConcatBetaTmaps(opt, funcFWHM, deleteIndBeta, deleteIndTmaps)
 
       saveAndRunWorkflow(matlabbatch, 'concat_betaImg_tMaps', opt, subID);
 
-      removeBetaImgTmaps(beta_maps, t_maps, deleteIndBeta, deleteIndTmaps);
+      removeBetaImgTmaps(beta_maps, t_maps, deleteIndBeta, deleteIndTmaps,ffxDir);
```
Comment (marcobarilari) [inline src/workflows/bidsConcatBetaTmaps.m:68]:
```suggestion
      removeBetaImgTmaps(beta_maps, t_maps, deleteIndBeta, deleteIndTmaps, ffxDir);
```

**Review by marcobarilari — APPROVED** [review]

**Comment by marcobarilari** [comment]
LGTM :) thanks

**Comment by CerenB** [comment]
no no it will not work... there's another dependency, funcFWHM

File: src/workflows/bidsConcatBetaTmaps.m [hunk 2]
```diff
@@ -65,15 +65,16 @@ function bidsConcatBetaTmaps(opt, funcFWHM, deleteIndBeta, deleteIndTmaps)
...
 
     end
   end
 
 end
 
-function removeBetaImgTmaps(beta_maps, t_maps, deleteIndBeta, deleteIndTmaps)
+function removeBetaImgTmaps(beta_maps, t_maps, deleteIndBeta, deleteIndTmaps, ffxDir)
```
Comment (marcobarilari) [inline src/workflows/bidsConcatBetaTmaps.m:75]:
```suggestion
function removeBetaImgTmaps(beta_maps, t_maps, deleteIndBeta, deleteIndTmaps, ffxDir, funcFWHM)
```

**Comment by marcobarilari** [comment]
right! there you go :)

**Comment by CerenB** [comment]
uupps sorry I did it already. I will not add your changes, also I've run mh_style fix as well! thanks anyways :)

**Comment by CerenB** [comment]
Actually, the goal of `subfunc` is not clear to me. Do we want to delete the `betas`? At the current form, we do not delete all the betas because `beta_maps` may not give the entire `beta_*.nii` created. We are only deleting the `betas` which were used to create `4D map`. 
So what do we want for thus deleting to do? All the `betas` to be clean? If yes, why not the following:

```
% delete maps
  if deleteIndBeta

    % delete all individual beta maps
    fprintf('Deleting individual beta-maps ...  ');
    delete(fullfile(ffxDir, ['beta_*','.nii']))
    fprintf('Done. \n\n\n ');

  end
```

Is there a reason why we need to be selectively deleting some betas but others? e.g. is the user use this `ffxDir` for some other beta storing? if not we can delete all the betas!

**Comment by marcobarilari** [comment]
Yes, what you are saying makes sense. Though have not tested yet that part so I was not aware of that 'incomplete' behavior.

I genuinely don't know what would be the best-case scenario here.

**Comment by CerenB** [comment]
@Remi-Gau any preference? if not, I'll merge it with the deletion implemented (pls see above).

**Comment by Remi-Gau** [comment]
I have no strong preference personally, mostly because I don't think I will ever want to delete the beta maps. 
So I will let you decide what the best usecase is. :-)  <---- look at me delegating the decision making process!!!! Yay!!!

**Pull request merged**